docs/en/getting-started/install.md

```diff
@@ -90,6 +90,16 @@ sudo clickhouse-client-$LATEST_VERSION/install/doinst.sh
 
 For production environments, it’s recommended to use the latest `stable`-version. You can find its number on GitHub page https://github.com/ClickHouse/ClickHouse/tags with postfix `-stable`.
 
+### From precompiled binaries for non-Linux OS {#from-binaries-non-linux}
+
+For non-Linux operating systems, ClickHouse builds are provided as a cross-compiled binary from the latest commit of the master branch.
+
+- macOS — `curl -O 'https://clickhouse-builds.s3.yandex.net/master/macos/clickhouse'`
```

Comment:
* let's also add clickable links to download from browser
* we'd better proxy those links through CDN so people Asia/America don't have to download from Moscow